src/database.js

```diff
@@ -39,7 +39,8 @@ export default class Database{
                                   }
 
                                   done();
-                                  callback(result.rows[0]);
+                                  var term = new Term(result.rows[0].id, result.rows[0].term, result.rows[0].definition, result.rows[0].tags.join([separator = ',']) || []);
```

Comment:
@dsagang, remember when readings docs. like https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split, the brackets are not part of the actual implementation. See some of the examples on the page on how to implement.